testdata/project-v3/main.go

```diff
@@ -122,6 +122,10 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "Laker")
 		os.Exit(1)
 	}
+	if err = (&crewv1.Captain{}).SetupWebhookWithManager(mgr); err != nil {
+		setupLog.Error(err, "unable to create webhook", "webhook", "Captain")
+		os.Exit(1)
+	}
```

Comment:
/hold

We should not duplicate it since the file will be re-created. 
Could we fix it to get it merged?

Reply:
Hi @camilamacedo86 With current behavior we can not avoid duplicating that. We can not force overwrite the main.go as the functionality is to overwrite the entire file, not the section. Also, main.go is edited by other apis as well and hence we can not do that. 

To remove the such duplicate scaffolding we need to fix [this](https://github.com/kubernetes-sigs/kubebuilder/blob/0a807f4e9428f7544a132fc2aa5ce4baa3510119/pkg/plugins/internal/machinery/scaffold.go#L315)

Reply:
would not possible if we use the --force option then not update the main.go?
Also, wdyt about we raise an issue in the repo to address this todo? It might give more visibility.

Reply:
Let me raise the issue and will try to work on it.
We can not skip the update on main.go because if user runs --force for the first time in webhook create command. It will not setup the webhook manager in the main.go.

Reply:
Good catcher.  I am ok with we move forward with it and work on the duplication/todo in a follow-up.  
/hold cancel